components/brave_wallet/browser/eth_tx_manager_unittest.cc

```diff
@@ -385,6 +385,34 @@ class EthTxManagerUnitTest : public testing::Test {
                                                    std::move(callback));
   }
 
+  void TestMakeERC1155TransferFromDataTxType(
+      const std::string& from,
+      const std::string& to,
+      const std::string& token_id,
+      const std::string& value,
+      const std::string& contract_address,
+      bool expected_success,
+      mojom::TransactionType expected_type) {
+    base::RunLoop run_loop;
+    eth_tx_manager()->MakeERC1155TransferFromData(
+        from, to, token_id, value, contract_address,
+        base::BindLambdaForTesting(
+            [&](bool success, const std::vector<uint8_t>& data) {
+              EXPECT_EQ(expected_success, success);
+              if (success) {
+                mojom::TransactionType tx_type;
+                std::vector<std::string> tx_params;
+                std::vector<std::string> tx_args;
+                ASSERT_TRUE(GetTransactionInfoFromData(ToHex(data), &tx_type,
+                                                       nullptr, nullptr));
```

Comment:
Check tx_params and tx_args here once we support it in eth_data_parser.cc

Reply:
Added those checks.